content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -211,11 +211,11 @@ yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
 systemctl enable --now kubelet
 ```
 
-  **Note:**
+  **Notes:**
 
   - Setting SELinux in permissive mode by running `setenforce 0` and `sed ...` effectively disables it.
     This is required to allow containers to access the host filesystem, which is needed by pod networks for example.
-    You have to do this until SELinux support is improved in the kubelet.
+  - Advanced users experienced in SELinux configuration may be able to keep SELinux enabled/enforcing but beware that it may require changes that are not covered by kubeadm support.
```

Comment:
Adjusting to [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices):
```suggestion
  - You can leave SELinux enabled if you know how to configure it but it may require settings that are not supported by kubeadm.
```